tests/integration/github/test_service.py

```diff
@@ -79,3 +83,18 @@ def test_project_create_in_the_group(self):
             repo=name_of_the_repo, namespace=namespace_of_the_repo
         )
         assert project.github_repo
+
+    def test_list_projects_with_user_input(self):
+        user = "packit"
+        service = GithubService(token=os.environ["GITHUB_TOKEN"])
+        projects = service.list_projects(user=user)
```

Comment:
```suggestion
        projects = self.service.list_projects(user=user)
```
you should be able to replace this since service is defined for the whole test class